xls/passes/rematerialization_pass.cc

```diff
@@ -964,6 +1009,11 @@ absl::StatusOr<bool> RematerializationReplacement(
 // TODO(zihao): 
 //   1. somehow integrate it into the current implementation
 //   2. check that if bugs exist
+// Comment(zihao): 
+//   It seems that this function has the equal ability with "cse_pass" to 
```

Comment:
Yeah, I was thinking the same thing when I read the earlier commit that introduced this. I think a `dry_run` on cse_pass would be less code for the same result.